### DISCUSSION TITLE
single repo, multiple projects

### DISCUSSION THREAD

**Discussion by sebalaini** [vercel/community#4648] [Help] [answered]
### Summary

Hello,

I was thinking about creating a monorepo to group 3 websites and then deploying them under different projects, currently, I have 3 repos, one for each, and in Vercel they are different projects, each one of them with a custom domain as well.

Is my understanding that if I were to create a monorepo I could deploy it in Vercel without problems, though which one would be the steps to override in the Build & Development Settings?

I guess that also my root package.json would have to include a build script for each website so that I could build and expose only the one I care about right?
something like `"build:a": "turbo run build --filter=a",`

### Steps to Reproduce

_

**Comment by amyegan** [comment, marked as answer]
Hey @sebalaini! We have some documentation and example projects to help you get started using Monorepos with Vercel. 😄  

- Docs: https://vercel.com/docs/monorepos
- Starter templates: https://vercel.com/templates/monorepos
- Another useful doc: Ignored Build Step has preset options so you can easily ignore unmodified projects when deploying a change. https://vercel.com/docs/projects/overview#ignored-build-step

**Comment by sebalaini** [comment]
hey thanks for the doc :) will dig into it.
The ignore build step is definitely very uselfull
